mapit_gb/management/commands/mapit_UK_add_slugs_to_local_authorities.py

```diff
@@ -0,0 +1,34 @@
+# This script is for a one off import of govuk slugs for local authorities
+
+import json
+import os.path
+
+from django.core.management.base import NoArgsCommand
+from mapit.models import Area, CodeType
+
+
+class Command(NoArgsCommand):
+    help = 'Assigns slugs from frontend to local authorities'
+
+    def handle_noargs(self, **options):
+        code_type, _ = CodeType.objects.get_or_create(
+            code='govuk_slug',
+            defaults={'description': 'Slug for use by GOV.UK'}
+        )
+
+        file_path = os.path.dirname(__file__) + '/../../data/authorities.json'
+        authorities = json.load(open(file_path))
+
+        for authority in authorities:
```

Comment:
Use [`items()`](https://docs.python.org/2/library/stdtypes.html#dict.items) when iterating over a dict if you need access to both the keys and the values:

```python
for slug, authority_details in authorities.items():
    gss_code = authority_details['gss']
```

...and then you have `slug` with a better name to use later on.